src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,266 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+
+/** md
+# Sequence Color Visualizer
+In this visualizer, the brightness of each circle represents how fast $1/a^n$, where a is a number in the sequence, approaches zero as n increases.
+Each prime number is assigned a color, and the color of each circle represents the color combination of its prime factors. 
+
+Press the "Left arrow key" to show/hide the color of each prime factor.
+Press the "Right arrow key" to refresh.
+ 
+ **/
```

Comment:
Another thought:  since a lot of sequences (including just the natural numbers) get to a bright full ball fairly quickly (in other words, all balls look pretty bright), maybe we could somehow "tune down" the brightness (e.g. run the given algorithm on a/10 or on log(a)) until we can see more a visual difference between, say natural numbers and squares?